docs/using_lagoon/environment_variables.md

```diff
@@ -78,3 +78,13 @@ If you want to define environment variables different per environment you can cr
 ### `.env` and `.env.defaults`
 `.env` and `.env.defaults` will act as the default values for environment variables if none other is defined. For example
 as default environment variables for Pull-Request environments (see [Worfklows](./workflows.md#pull-requests)).
+
+## Special Environment Variables
+
+### `PHP_ERROR_REPORTING`
```

Comment:
@cdchris12 
can you please add this documentation here: https://lagoon.readthedocs.io/en/latest/using_lagoon/docker_images/php-fpm/

Reply:
Added in d9c6f3616230e85408c79cfc8fc1692c44cf9ede . Would you like to remove this block from `docs/using_lagoon/environment_variables.md` as well? @Schnitzel